UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.LoadTests/TestCase/ESSPTUsingLiveLogs.js

```diff
@@ -0,0 +1,122 @@
+import { sleep } from 'k6';
+import { scenario } from 'k6/execution';
+import { textSummary } from "https://jslib.k6.io/k6-summary/0.0.1/index.js";
+import { htmlReport } from "https://raw.githubusercontent.com/benc-uk/k6-reporter/main/dist/bundle.js";
+
+const essLogsFile = JSON.parse(open('../TestData/essLogs.json')); //Add path of Json file containing Azure logs of ESS (peak load hour) 
+const essAPI = require('../Scripts/ReplayRequest.js');
+const logParser = require('../Helper/LogParser.js');
+const testHelper = require('../Helper/LoadStages.js');
+const testStages = testHelper.getLoadStages(essLogsFile);
+const stageTime = '1m';
+
+export const options = {
+    scenarios: {
+    'hardcoded-executor': {
+      executor: 'ramping-arrival-rate',
+      preAllocatedVUs: 180, 
```

Comment:
Why 180 VUs?  You should probably change this to something smaller unless you need it, just because it is confusing to the reader.

Reply:
To handle spiked request rate per minute according to the logs data, we can reduce it depending up to server performance and VM configuration.